.storybook/middleware.js

```diff
@@ -0,0 +1,12 @@
+const { createProxyMiddleware } = require('http-proxy-middleware');
```

Comment:
Don't forget to load environments from your dotenv.

```ts
const dotenv = require('dotenv');
dotenv.config();
```

Reply:
Thanks!